docs/integration/kubernetes.md

```diff
@@ -0,0 +1,288 @@
+
+
+
+
+![alt_text](https://sematext.com/wp-content/uploads/2020/03/Kubernetes-Pod-Overview.png "Sematext Monitoring")
+
+
+
+### Helm Chart
+
+To start monitoring Kubernetes with Sematext install the Sematext Agent. The easiest way to do that is with a Helm chart. It’s available in the official charts repo and it will install to all nodes in your cluster. To install it run the following command:
+
+
+
+
+```
+helm install --name sematext-agent \
+  --set containerToken=<YOUR_CONTAINER_TOKEN> \
+  --set infraToken=<YOUR_INFRA_TOKEN> \
+  --set logsToken=<YOUR_LOGS_TOKEN> \
+  --set region=<"US" or "EU"> \
+  stable/sematext-agent
+```
+
+
+
+Check out [github](https://github.com/helm/charts/blob/master/stable/sematext-agent/README.md) for more details.
+
+
+### Sematext Operator
+
+You can also install Sematext Operator using this command:
+
+
+
+
+```
+kubectl apply -f https://raw.githubusercontent.com/sematext/sematext-operator/master/bundle.yaml
+```
+
+
+
+
+After the installation has finished you can create the SematextAgent resource that deploys the agent to all the nodes in your cluster.
+
+
+
+
+```
+apiVersion: sematext.com/v1alpha1
+kind: SematextAgent
+metadata:
+  name: sematext-agent
+spec:
+  region: <"US" or "EU">
+  containerToken: YOUR_CONTAINER_TOKEN
+  logsToken: YOUR_LOGS_TOKEN
+  infraToken: YOUR_INFRA_TOKEN
+```
+
+
+
+For those looking for a more hands-on approach, there’s a [manual installation procedure](https://sematext.com/docs/agents/sematext-agent/kubernetes/installation/#manual-installation) with `kubectl`.
+
+
+## Shipping Kubernetes logs to Sematext
+
+
+Due to its nature, Kubernetes can be difficult to debug and without proper tooling this process will take a lot longer than it has too. Sematext helps you shed light on what caused the anomaly that led to the crash.
+
+
+
+
+
+![alt_text](https://sematext.com/docs/images/logs/logsene-ui.png "Sematext Logs")
+
+
+To configure Kubernetes log shipping we’re going to use Helm.
+
+
+### Helm
+
+To install Logagent with Helm you’ll need to run the following command:
+
+```
+
+helm install st-logagent \
+
+  --set logsToken=860d2539-3e00-4b4a-969d-13b4a4f4b8e7 \
+
+  --set region=EU \
+
+  stable/sematext-agent
+
+```
+
+Deleting Logagent can be done with:
+
+```
+
+helm delete st-logagent
+
+```
+
+
+If you are looking to use a different type of integration you can check out this [page](https://sematext.com/docs/logagent/installation-docker/#kubernetes-and-openshift).
+
+
+## Kubernetes Metrics 
+
+Container and Kubernetes metrics are collected along with labels and tags, which are exposed in the UI to allow slicing and dicing and building of custom dashboards.
```

Comment:
I think we don't have to list container metrics again. They are already explained in Docker integrations. 
In this sprint I have a task for documenting kubernetes metrics fields. @johndemian When are you merging this to master?